src/cascadia/TerminalControl/TermControl.xaml

```diff
@@ -55,8 +55,6 @@
         -->
 
         <x:Double x:Key="ScrollBarSize">16</x:Double>
-        <CornerRadius x:Key="ScrollBarCornerRadius">0</CornerRadius>
-        <CornerRadius x:Key="ScrollBarThumbCornerRadius">3</CornerRadius>

```

Comment:
@zadjii-msft I've opted to simplify our customization a bit so that we only need to add or delete whole lines in the future. I've marked the 3 places that got changed with `<!-- -->`. Works great (200% scale):

![image](https://user-images.githubusercontent.com/2256941/218589106-a039d863-ac19-4cd0-81e8-0b315f158d85.png)